app/templates/components/insights-glance.hbs

```diff
@@ -0,0 +1,27 @@
+<h3 class="insights-glance__title">{{title}}</h3>
+<div class="insights-glance__center">
+  <div class="insights-glance__stat">
+    {{#if isLoading}}
+      {{loading-indicator}}
+    {{else}}
+      {{statistic}}
+    {{/if}}
+  </div>
+  {{#if delta}}
+    <div
+      class="insights-glance-delta"
+      data-dir="{{if (gte delta 0) '+' '-'}}"
+      title="{{deltaTitle}}"

```

Comment:
It's generally not necessary to wrap handlebars placeholders into quotes, compiler takes care of that.

Reply:
Ahh nice, good to know :) Change made